manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -149,7 +149,7 @@ spec:
           - name: CLUSTER_ID
             value: "{{ $.Values.global.multiCluster.clusterName | default `Kubernetes` }}"
           - name: EXTERNAL_ISTIOD
-            value: "{{ $.Values.global.externalIstiod | default "false" }}"
+            value: "false"
```

Comment:
Changed as discussed here: https://github.com/istio/istio.io/pull/9679#discussion_r626077176

Reply:
why set it at all then? its already the default in code

Reply:
Good point, wasn't thinking :(
Fixed.